Dockerfile

```diff
@@ -6,83 +6,58 @@ RUN /tmp/build-pgloader && rm /tmp/build-pgloader
 FROM ruby:2.6-stretch
 MAINTAINER operations@openproject.com
 
-ENV NODE_VERSION "10.15.0"
-ENV BUNDLER_VERSION "2.0.2"
+# Allow platform-specific additions. Valid values are: on-premise,cloud
+ARG PLATFORM=on-premise
+# Use OAuth token in case private gems need to be fetched
+ARG GITHUB_OAUTH_TOKEN
+ARG DEBIAN_FRONTEND=noninteractive
+
+ENV NODE_VERSION="12.18.3"
+ENV BUNDLER_VERSION="2.0.2"
```

Comment:
I suggest we use `2.1.4` here to keep up with what's defined in the `Gemfile.lock` already.